src/explore-education-statistics-admin/src/pages/release/ManageReleasePageContainer.tsx

```diff
@@ -23,9 +27,18 @@ const ManageReleasePageContainer = ({
   const { publicationId, releaseId } = match.params;
 
   const [publication, setPublication] = useState<BasicPublicationDetails>();
+  const [releasePublicationStatus, setReleasePublicationStatus] = useState<
+    ReleasePublicationStatus
+  >();
 
   useEffect(() => {
-    service.getBasicPublicationDetails(publicationId).then(setPublication);
+    Promise.all([
```

Comment:
Not necessary, but again, we could use `useAsyncRetry` here.

Reply:
Happy to do so!